doc/plugins.md

```diff
@@ -274,48 +276,56 @@ The list of plugins:
 
 ## List of presets
 
-See [npm search][npm-preset-search] or [github search][github-preset-search]
-for available and often inspirational presets.
+Use [GitHub search][github-preset-search] to find available and often
+inspirational presets.
 
 ## List of utilities
 
-See [**mdast**][mdast-util] for a list of utilities for working with the syntax
+See [mdast][mdast-util] for a list of utilities that work with the syntax
 tree.
-See [`unist`][unist-util] for other utilities which work with **mdast**
-nodes, too.
-Finally, see [**vfile**][vfile-util] for a list of utilities working with
-virtual files.
+See [unist][unist-util] for other utilities which work with **mdast** and other
+syntax trees too.
+Finally, see [vfile][vfile-util] for a list of utilities working with virtual
+files.
 
-## Using plugins
+## Use plugins
 
 To use a plugin programmatically, call the [`use()`][unified-use] function.
-
 To use plugin with `remark-cli`, pass a [`--use` flag][unified-args-use] or
 specify it in a [configuration file][config-file-use].
 
-## Creating plugins
-
-Have an idea for a plugin?
-Post it in [ideas][] and make it happen!
+## Create plugins
 
 To create a plugin, first read up on the [concept of plugins][unified-plugins].
 Then, read the [guide on “Creating a plugin with unified”][guide].
 Finally, take one of existing plugins, which looks similar to what you’re about
 to make, and work from there.
-If you get stuck, [ideas][], [issues][], and [discussions][] are good places to
-get help.
-
-You should pick a name prefixed by `'remark-'`, such as `remark-lint`.
+If you get stuck, [discussions][] is a good place to get help.
 
+You should pick a name prefixed by `'remark-'` (such as `remark-lint`).
 **Do not use the `remark-` prefix** if the thing you create doesn’t work with
 `remark().use()`: it isn’t a “plugin” and will confuse users.
 If it works with mdast, use `'mdast-util-'`, if it works with any unist tree,
 use `unist-util-`, and if it works with virtual files, use `vfile-`.
 
+Use default exports to expose plugins from your packages, add `remark-plugin`
+keywords in `package.json`, and add a `remark-plugin` topic to your repo on
+GitHub.
+
```

Comment:
Perhaps you could invite users to add their own plugins or plugins they know about to the list here.

Reply:
great idea, done!